awx/main/notifications/grafana_backend.py

```diff
@@ -27,7 +27,10 @@
 DEFAULT_APPROVAL_DENIED_MSG = CustomNotificationBase.DEFAULT_APPROVAL_DENIED_MSG
 DEFAULT_APPROVAL_DENIED_BODY = CustomNotificationBase.DEFAULT_APPROVAL_DENIED_BODY
 
+logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
I think these 3 lines are left over from your troubleshooting?  They should be removed.

Reply:
Please remove debug code